src/script/lua_api/l_item.cpp

```diff
@@ -535,16 +535,22 @@ int ModApiItemMod::l_register_item_raw(lua_State *L)
 	// Register item definition
 	idef->registerItem(def);
 
+	if (def.type != ITEM_NODE)
+		return 0;
```

Comment:
having the early return here for `ITEM_NODE` implies that a node type for register_item_raw is somehow wrong, I'd keep that here.

Reply:
ok